edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Diagnostics/Microsoft.Azure.Devices.Edge.Agent.Diagnostics.csproj

```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.23.1" />
+    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.23.1-NestedEdge" />
```

Comment:
Your changes in the SDK will go in their next release, right? aka we won't have to maintain a separate sdk version for too long?

Reply:
I don't think the timing is locked down yet, since I've updated the REST API version in the SDK, they can't release this change until the IoT Hub changes from Anca and David go public.

Reply:
We should rebase the private SDK bits with the changes from 1.26.0 version of the SDK